src/controllers/v1/modules.js

```diff
@@ -0,0 +1,81 @@
+const common = require('@constants/common')
+const modulesService = require('@services/modules')
+
+module.exports = class modules {
+	/**
+	 * create modules
+	 * @method
+	 * @name create
+	 * @param {Object} req - request data.
+	 * @returns {JSON} - modules creation object.
+	 */
+
+	async create(req) {
```

Comment:
These APIs should only be accesssed by system admin right? Can you add that layer of validation Refer admin APIs.